mPower2/MotorControl/MCTOverviewStepViewController.swift

```diff
@@ -0,0 +1,137 @@
+//
+//  MCTOverviewStepViewController.swift
+//  MotorControl
+//
+//  Copyright © 2018 Sage Bionetworks. All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without modification,
+// are permitted provided that the following conditions are met:
+//
+// 1.  Redistributions of source code must retain the above copyright notice, this
+// list of conditions and the following disclaimer.
+//
+// 2.  Redistributions in binary form must reproduce the above copyright notice,
+// this list of conditions and the following disclaimer in the documentation and/or
+// other materials provided with the distribution.
+//
+// 3.  Neither the name of the copyright holder(s) nor the names of any contributors
+// may be used to endorse or promote products derived from this software without
+// specific prior written permission. No license is granted to the trademarks of
+// the copyright holders even if such marks are included in this software.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE
+// FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+// DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+// CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+// OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+
+import Foundation
+
+open class MCTOverviewStepViewController : RSDOverviewStepViewController {
+    
+    /// The image views to display the icons on.
+    @IBOutlet
+    open var iconImages: [UIImageView]!
+    
+    /// The labels to display the titles of the icons on.
+    @IBOutlet
+    open var iconTitles: [UILabel]!
+    
+    @IBOutlet
+    open var infoButton: UIButton!
+    
+    @IBOutlet
+    open var scrollView: UIScrollView!
+    
+    /// Overrides viewWillAppear to add an info button, display the icons, to save
+    /// the current Date to UserDefaults, and to use the saved date to decide whether
+    /// or not to show the full task info or an abbreviated screen.
+    override open func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        // Add the info button.
+        
```

Comment:
Can you add a comment that we are assuming that there are either 1 or 3 icons and that other implementations would need to use a `UICollectionView`? Thanks